content/en/blog/_posts/2022-12-06-kubernetes-1.25-blog.md

```diff
@@ -0,0 +1,205 @@
+
+Kubernetes v1.26 drops support for CRI `v1alpha2`. That [removal](https://github.com/kubernetes/kubernetes/pull/110618) will result in the kubelet not registering the node if the container runtime doesn't support CRI `v1`. This means that containerd minor version 1.5 and older will not be supported in Kubernetes 1.26; if you use containerd, you will need to upgrade to containerd version 1.6.0 or later **before** you upgrade that node to Kubernetes v1.26. Other container runtimes that only support the `v1alpha2` are equally affected: if that affects you, you should contact the container runtime vendor for advice or check their website for additional instructions in how to move forward.
+
+### Progress on the CSI migration process
+
+Following the GA of the [core Container Storage Interface (CSI) Migration](https://github.com/kubernetes/enhancements/tree/master/keps/sig-storage/625-csi-migration) feature in the previous release, CSI migration is an on-going effort that we've been working on for a few releases now, and this release continues to add (and remove) features aligned with the migration's goals.
+
+#### Delegate FSGroup to CSI Driver graduated to Stable
+
+This feature allows Kubernetes to [supply the pod's `FSGroup` to the CSI driver when a volume is mounted](https://github.com/kubernetes/enhancements/issues/2317) so that the driver can utilize mount options to control volume permissions. Previously, the kubelet would always apply the `fsGroup`ownership and permission change to files in the volume according to the policy specified  in the `Pod.spec.securityContext.fsGroupChangePolicy` field. Starting with this release, CSI drivers have the option to apply the `fsGroup` settings during attach or mount time of the volumes.
+
+#### CSI migration for Azure File and vSphere graduated to Stable
+
+Both the [vSphere](https://github.com/kubernetes/enhancements/issues/1491) and [Azure](https://github.com/kubernetes/enhancements/issues/1885) in-tree driver migration to CSI have graduated to Stable. You can find more information about them in the [vSphere CSI driver](https://github.com/kubernetes-sigs/vsphere-csi-driver) and [Azure File CSI driver](https://github.com/kubernetes-sigs/azurefile-csi-driver) repositories.
+
+#### In-tree GlusterFS driver removal
+
+Already deprecated in the v1.25 release, the in-tree GlusterFS driver was [removed](https://github.com/kubernetes/enhancements/issues/3446) in this release.
+
+#### In-tree OpenStack Cinder driver removal
+
+This release removed the deprecated in-tree storage integration for OpenStack (the `cinder` volume type). You should migrate to external cloud provider and CSI driver from https://github.com/kubernetes/cloud-provider-openstack instead. For more information, visit [Cinder in-tree to CSI driver migration](https://github.com/kubernetes/enhancements/issues/1489).
```

Comment:
I wonder if we shouldn't delete that, since we already announced that in our last blog post.

Reply:
Both removals, or just one? We could remove both in the interest of being more concise (they are all listed in the Removals section). I've kept them since they are part of the CSI migration theme and since I grouped them, the impact is smaller. If we remove them we should reflect that in the previous proposal of how many removals we want to highlight at the very start - in that case, we would just have the CRI API one.

Reply:
We should leave these in. Don't assume readers read the previous blog article that announced upcoming removals.

Reply:
+1 removing both